packages/itwin/grouping-mapping-widget/src/widget/components/GroupsVisualization.tsx

```diff
@@ -202,6 +213,23 @@ export const GroupsVisualization = ({
     ),
   ].flat(), [groups, hideSingleGroupWrapper, isLoadingQuery, showGroup, showGroupColor]);
 
+  const overlappedAlert = useMemo(() =>
+    overlappedElementsInfo.size > 0 && isAlertClosed && showGroupColor && !isVisualizing ?
+      <Alert
+        onClose={() => setIsAlertClosed(false)}
+        clickableText={isAlertExpanded ? "Less Details" : "More Details"}
+        clickableTextProps={{ onClick: () => setIsAlertExpanded(!isAlertExpanded) }}
+      >
+        Overlapping elements are colored <Text className="gmw-red-text">red</Text> in the viewer.
+        {isAlertExpanded ? (
+          <>
+            <br />
+            To get overlap info in details, click the &ldquo;Overlap Info&rdquo; in the menu icon adjacent to the groups.
```

Comment:
The wording is a bit weird. "Click the Overlap Info in the **menu icon** ... ". How can something be inside the "menu icon"?

Reply:
![image](https://github.com/iTwin/viewer-components-react/assets/26608199/43cb8a29-b89e-418f-a414-460f891f2586)